core/arch/arm/plat-sam/platform_config.h

```diff
@@ -41,6 +53,18 @@
 #error "LPAE not supported"
 #endif
 
+#if defined(PLATFORM_FLAVOR_sama7g54_ek)
+#define CONSOLE_UART_BASE   (FLEXCOM3_BASE_ADDRESS + 0x200)
+#define SFR_BASE            (SFR_BASE_ADDRESS)
```

Comment:
can remove parentheses

Reply:
The parentheses in definition for `SFR_BASE` removed, thank you.